patientsearch/api.py

```diff
@@ -30,7 +31,14 @@
 from patientsearch.extensions import oidc
 from patientsearch.jsonify_abort import jsonify_abort
 
-api_blueprint = Blueprint("patientsearch-api", __name__)
+api_blueprint = Blueprint("patientsearch-api", __name__, cli_group=None)
+
+
+@api_blueprint.cli.command("justtesting", help="yes, need help")
+@click.argument("name", required=True)
+def justtesting(name):
+    click.echo(f"hello {name}")
+    audit_entry("in justtesting()")
```

Comment:
was this helpful in testing, or leftover?...

Reply:
💯